README.md

```diff
@@ -167,6 +167,8 @@ These plugins can optionally execute after the `run` command for Accumulo and Fl
 
 * `spark` - Install Apache Spark and start Spark's History server
   * [Spark HistoryServer](http://localhost:18080/)
+* `accumulo-proxy` - Starts an [Accumulo Proxy] which enables Accumulo clients in other languages.
+  * Follow these [instructions][client-instruct] to create a Python Accumulo client.
```

Comment:
I would leave the Python client instructions link out of the Uno instructions.  I think the pointer to the proxy Readme on the previous line is sufficient.

Reply:
Sounds good to me